bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml

```diff
@@ -186,6 +186,7 @@ limitations under the License.
                     -->
                     <exclude>com.google.cloud.bigtable.hbase*.**</exclude>
                     <exclude>com.google.cloud.bigtable.metrics.**</exclude>
+                    <exclude>com.google.cloud.bigtable.hbase.wrappers.veneer.metrics.**</exclude>
```

Comment:
here and all of the other shaded poms: make sure to exclude the artifact as well